clients/vault/src/system.rs

```diff
@@ -425,10 +427,24 @@ impl VaultService {
 				run(wallet::listen_for_new_transactions(
 					self.stellar_wallet.get_public_key(),
 					self.stellar_wallet.is_public_network(),
-					issue_hashes_vec.clone(),
 					watcher.clone(),
-					wallet::types::TxFilter, /* todo: change with a filter specific to the issue
-					                          * request? */
+					issue_map.clone(),
+					IssueFilter,
+				)),
+			),
+			(
+				"Listen for Issue Requests",
+				run(issue::listen_for_issue_requests(
+					self.spacewalk_parachain.clone(),
+					secret_key.clone(),
+					issue_map.clone(),
+				)),
+			),
+			(
```

Comment:
Should we also run the `listen_for_executed_issues` here?

Reply:
😬 oh yes, i forgot. I'll add it right away.